### PR TITLE
Create .zappr.yaml

### PR DESCRIPTION
**Pull request by harti2006**

### CODE REVIEW

**Comment by harti2006** [comment]
:+1:

File: .zappr.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# for github.com
+approvals:
+  groups:
+    zalando:
+      minimum: 2
```
Comment (hjacobs) [inline .zappr.yaml:5]: Do we really need two approvers?

File: .zappr.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      from:
+        orgs:
+          - "zalando"
+
+  # Allow last committer / PR creator to approve as well.
+  # This will probably become the default in zappr.
+  ignore: none
+X-Zalando-Team: "torch"
```
Comment (hjacobs) [inline .zappr.yaml:13]: What is this for?

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**